policy-release/action.yml

```diff
@@ -14,57 +14,78 @@ inputs:
   GITHUB_TOKEN:
     required: true
     type: string
+  policy-working-dir:
+    description: "working directory of the policy. Useful for repos with policies in folders"
+    required: false
+    type: string
+    default: .
+  policy-version:
+    description: "release version of the policy. Useful for repos with policies in folders"
+    required: false
+    type: string
+
 runs:
   using: "composite"
   steps:
-    -
-      name: Login to GitHub Container Registry
+    - id: calculate-version
```

Comment:
> We should also use a policy version variable here to make this agnostic.

It's not as simple, as the GH release needs to match the git tag that it was triggered with, *only* when the trigger of this workflow was indeed from a tag (e.g: `v0.1.0`) instead of the "monorepo tag" (`policy/v0.1.0` edited into `0.1.0`).

Instead of again recalculating the version from the "monorepo tag", as we do in the monorepo workflow defition, here I just add a new, non-required `policy-version` input, that when present changes the behaviour.

I can take this out of the action, but that means passing that information through the reusable workflows, and potentially editing workflows of the policies I have, as that breaks backwards compatibility.

Reply:
After a tag, I think you are right and the `calculate_version` step can be dropped. Let me see.

Reply:
Added some comments that remove the calculate-version from the `policy-release` action, and expect this calculation to have been taken place in the workflows, by now making `policy-release.inputs.policy-version` required.
Hence, amended the workflows to always calculate-version and pass this value down the chain.